.github/workflows/test_web.yml

```diff
@@ -92,11 +92,16 @@ jobs:
         working-directory: web
         run: npm test
 
+      - name: Set EDGEDRIVER_VERSION environment variable
```

Comment:
I think a comment explaining what this is and why it's there and what it's ultimately achieving, would be appreciated - just for future maintainability and bus factor and similar reasons...